src/platform.js

```diff
@@ -0,0 +1,8 @@
+class Platform {
+    static setNativeModule(module) {
```

Comment:
If the only purpose of `setNativeModule()` is to set the platform information it should be renamed to `setPlatformInformation()`. The name `setNativeModule` is very generic and it does not describes what the function does.